samples/avr/crosstool.config

```diff
@@ -1,4 +1,5 @@
 CT_CONFIG_VERSION="4"
 CT_ARCH_AVR=y
+# CT_CC_GCC_ENABLE_DEFAULT_PIE is not set
```

Comment:
I think it would be better to add `&& ! ARCH_AVR && ! ARCH_PRU` to the depends in  [config/cc/gcc.in](https://github.com/crosstool-ng/crosstool-ng/blob/master/config/cc/gcc.in#L200). That way any custom AVR configs won't need updating

Reply:
Thanks. Fixed and updated the pull request.